Templates/zOSNativeGitlabCIPipeline/.gitlab-ci.yml

```diff
@@ -0,0 +1,470 @@
+        - if: $CI_COMMIT_BRANCH =~ /release.*/ && $pipelineType == "release"
+
+    dependencies: []
+    artifacts:
+        name: “productionDeploymentReport-${CI_PIPELINE_ID}"
+        when: always
+        paths:
+            - "deployPkgDir/deploy-production/evidences/evidence.yaml"
+            - "deployPkgDir/deploy-production/deployment-report.html"
+    script: |
+            wazideploy-evidence.sh -w ${uniqueWorkspaceId} -l deploy-production/evidences/evidence.yaml -o deploy-production/deployment-report.html
+            RC=$?
+            if [ ${RC} -eq 0 ]; then
+                echo [INFO] Generated deployment evidence.        
+            else
+                echo [ERROR] Failed to generate deployment evidence file. Exit code: ${RC}.
+                exit ${RC}
+            fi
+
+Create production version:
+    needs: ["1-Deploy to Production"]
+    stage: Finalize
+    tags: [zos-native]
+    rules:
+        - if: $CI_COMMIT_BRANCH == "main" && $pipelineType == "release"
+        - if: $CI_COMMIT_BRANCH =~ /release.*/ && $pipelineType == "release"
+    script: |
+            echo [INFO] "Start creating production version job"
+
+            stringOut=$(computeReleaseVersion.sh -w ${uniqueWorkspaceId} -a ${application} -b ${CI_COMMIT_REF_NAME} -r ${releaseType})
+            RC=$?
+
+            if [ ${RC} -eq 0 ]; then
+                export releaseVersion=$(echo "${stringOut}" | grep "The next release version:" | awk -F "rel-" '{print $2}' | awk -F ". rc=" '{print $1}' | xargs)
+                export oldVersionTag=$(echo "${stringOut}" | grep "Baseline reference:"  | awk -F "refs/tags/rel-" '{print $2}' | xargs)
+
+                oldVersionTag="rel-${oldVersionTag}"
+                echo [INFO] Current version tag: ${oldVersionTag}
+                releaseVersion="rel-${releaseVersion}"
+                echo [INFO] Release version tag: ${releaseVersion}
+
+                cd ${uniqueWorkspaceId}/${application}
+
+                export remoteFound=`git remote | grep gitlab_origin | wc -l`
+                if [ $remoteFound -eq 1 ]; then
+                    git remote remove gitlab_origin
+                fi
+                git remote add gitlab_origin ${CI_SERVER_PROTOCOL}://Automation:${AutomationToken}@${CI_SERVER_HOST}:${CI_SERVER_PORT}/${CI_PROJECT_PATH}.git
+
+
+                # Update baselineReference
+                baselineReferenceFile=$(find ${uniqueWorkspaceId} -name baselineReference.config)
+                if [ ${CI_COMMIT_BRANCH} == "main" ]; then
+                    echo [INFO] Update baselineReference.config file to set baseline references for the branches main and release/rel-${releaseVersion}
+                    cat ${baselineReferenceFile} | sed "s/main=refs\/tags\/${oldVersionTag}/main=refs\/tags\/${releaseVersion}/" > ${baselineReferenceFile}.new
+                    iconv -f ISO8859-1 -t IBM-1047 ${baselineReferenceFile}.new > ${baselineReferenceFile}
```

Comment:
Is this an issue with the BPXK_AUTOCVT being on or off? This runs on USS - why do we need to do the `iconv` here?